src/NBitcoin/ChainBase.cs

```diff
@@ -161,6 +213,11 @@ public ChainedBlock FindFork(BlockLocator locator)
             return this.FindFork(locator.Blocks);
         }
 
+        /// <summary>
+        /// Enumerate chain after given block hash to genesis block.
+        /// </summary>
+        /// <param name="blockHash">Block hash to enumerate after.</param>
+        /// <returns>Enumeration of chain after given block hash.</returns>
         public IEnumerable<ChainedBlock> EnumerateAfter(uint256 blockHash)
```

Comment:
as mentioned before, these enumeration methods, if they could be checked whether they are used and whether their use is proper - they can be pretty heavy, so we should be careful

not for this PR of course